deploy/images/db/ebi.sql

```diff
@@ -1,85 +1,47 @@
+CREATE SCHEMA local_ega_ebi;
+
+SET search_path TO local_ega_ebi;
 
 -- Special view for EBI Data-Out
-CREATE VIEW local_ega.ebi_files AS
-SELECT id                                       AS file_id,
-       stable_id                                AS file_name,
-       archive_file_reference                     AS file_path,
-       archive_file_type                          AS file_type,
-       archive_file_size                          AS file_size,
-       archive_file_checksum                      AS unencrypted_checksum,
-       archive_file_checksum_type                 AS unencrypted_checksum_type,
-       header                                   AS header,
-       created_by                               AS created_by,
-       last_modified_by                         AS last_updated_by,
-       created_at                               AS created,
-       last_modified                            AS last_updated
+CREATE VIEW local_ega_ebi.file AS
+SELECT stable_id                                AS file_id,
+       archive_file_reference                   AS file_name,
+       archive_file_reference                   AS file_path,
+       reverse(split_part(reverse(submission_file_path::text), '/'::text, 1)) AS display_file_name,
+       archive_file_size                        AS file_size,
+       NULL::text                               AS checksum,
+       NULL::text                               AS checksum_type,
+       archive_file_checksum                    AS unencrypted_checksum,
+       archive_file_checksum_type               AS unencrypted_checksum_type,
+       status                                   AS file_status,
+       header                                   AS header
 FROM local_ega.main
 WHERE status = 'READY';
 
-
--- Relation File <-> Index File
-CREATE TABLE local_ega.index_files (
-       id       SERIAL, PRIMARY KEY(id), UNIQUE (id),
-
-       file_id  INTEGER NOT NULL REFERENCES local_ega.main (id) ON DELETE CASCADE,
-
-       index_file_reference      TEXT NOT NULL,     -- file path if POSIX, object id if S3
-       index_file_type           local_ega.storage  -- S3 or POSIX file system
-);
-
 -- Relation File EGAF <-> Dataset EGAD
-CREATE TABLE local_ega.file2dataset (
+CREATE TABLE local_ega_ebi.filedataset (
        id       SERIAL, PRIMARY KEY(id), UNIQUE (id),
-
        file_id     INTEGER NOT NULL REFERENCES local_ega.main (id) ON DELETE CASCADE, -- not stable_id
```

Comment:
I think here should be the `stable_id`, because when there is a request for a file via DataEdge for a file it is done via the `stable_id` e.g. 
```
curl -G -H "Authorization: Bearer $TOKEN" https://localhost:9090/files/EGA798762706
```

DataEdge makes a request in `FileInfoServiceImpl.java` to `getFilePermissionsEntity(String stableId)` and uses that stableID to call `FileDatabase/file/{stableId}/datasets` which in return looks into the view `local_ega_ebi.file_dataset` which gets data from this table - hopefully I got this right.

Long story short - we will be exposing internal file ids and we won't be able to retrieve a file with its stable_id from DataEdge if we do it like this.

Reply:
I assume some additional logic might need implementing in `ega-data-api` to verify that the file with that stableID is there in `ebi_files` ...

Reply:
The download_log and event log are now in local_ega_download.requests local_ega_download.success and local_ega_download.errors.

So the table links with the file_id of the main table but the file_dataset view (not filedataset table) has the stable_id.

To test it you with ega-data-api with current master you have to do 2 things:
* modify the code in 
the lines: 
from
```
    new SqlOutParameter("vault_path", Types.VARCHAR),
    new SqlOutParameter("valut_type", Types.OTHER),
```
to
```
    new SqlOutParameter("archive_path", Types.VARCHAR),
    new SqlOutParameter("archive_type", Types.OTHER),
```

Start ega-data-api with these profiles:

dataedge: no-oss,logger-log,add-user-ip-headers
res: no-oss,LocalEGA,log-transfer,repo-logger,db-repo-logger

Also for res you have to configure the database connection:
```
spring.datasource.url= jdbc:postgresql://localhost:5432/lega
spring.datasource.username=lega_out
spring.datasource.password=db_password
spring.jpa.properties.hibernate.default_schema = local_ega_ebi

## DataSource common settings
# Keep the connection alive if idle for a long time (needed in production)
spring.datasource.hikari.test-while-idle = true
spring.datasource.hikari.validation-query = SELECT 1
spring.datasource.hikari.initial-size = 3
spring.datasource.hikari.maximum-pool-size = 8
spring.datasource.hikari.minimum-idle = 1
```
About "Also not 100% sure how the new structure will satisfy the need for audit mentioned here: EGA-archive/ega-data-api#10 (comment)" Not sure I see what you mean. We can talk on slack if you want.

Reply:
> So the table links with the file_id of the main table but the file_dataset view (not filedataset table) has the stable_id.

Yes you are right I missed that.
Will try to test ASAP.